### PR TITLE
[DOC-159] DS v5.2 Release Updates

### PR DESCRIPTION
**Pull request by MrEdData**
- Links on the Ed-Fi Data Standard page is updated to reflect the new release DS v5.2
- What's New - v5.2 link is added to the What's New Overview page
- What's New - v5.2 sub-page is added with details of the DS model updates from v5.1 to v5.2

### CODE REVIEW

File: dataStandard_versioned_docs/version-5/whats-new/whats-new-v52.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+* Education organizations collaborate with the assessment vendor to determine the time frame and guidelines for administering an assessment.
+* The specifics of assessment participation and delivery are outlined.
+* Students are registered for the administration of the assessment.
+* The assessment vendor receives the registration list of students along with selected student information.
+
+#### Model Details
+
+To allow community members track data in the Ed-Fi UDM, Assessment Registration domain is created with the following entities and associations:
+
+<u>_AssessmentAdministration_:</u> It is to store data for the administration of an assessment. It has references to AssigningEducationOrganization and Assessment entities, and tracks information relating to AdministrationIdentifier, AssessmentAdministrationPeriod and AssessmentBatteryPart.
```
Comment (stephenfuqua) [inline dataStandard_versioned_docs/version-5/whats-new/whats-new-v52.md:50]: We usually don't utilize underlining in this document, only italics or bold. We don't have to change it right now, but I recommend changing in the future. To be consistent with other areas of the documentation, I would use bold only, without italics or underline.

File: dataStandard_versioned_docs/version-5/whats-new/whats-new-v52.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+
+<u>_AssessmentBatteryPart_:</u> The parts organized for administering an assessment which together provide a comprehensive assessment of the students.
+StudentAssessmentRegistrationBatteryPartAssociation: The association to the part(s) of the assessment battery that the student is to be tested for this administration of the assessment.
+
+<u>_StudentEducationOrganizationAssessmentAccommodation_:</u> The accommodation(s) required or expected for administering assessments as determined by the education organization.
+
+The UML diagram below illustrates the model for the Assessment Registration domain with its entities, attributes and associations.
+
+![Assessment Registration Domain](https://edfidocs.blob.core.windows.net/$web/img/reference/data-standard/v52/UML_AssessmentRegistrationDomain_DSv52.jpg)
+[_Click here for a larger view of the diagram_](https://edfidocs.blob.core.windows.net/$web/img/reference/data-standard/v52/UML_AssessmentRegistrationDomain_DSv52.jpg)
```
Comment (stephenfuqua) [inline dataStandard_versioned_docs/version-5/whats-new/whats-new-v52.md:63]: Best to have a line break between lines 62 and 63.  The "larger version" is typically a large image. Many of the images previously created from Visio are very large, much larger than what is needed for display in a web page. I see that this file is 375 KB, which is big but not gigantic (previously, some of the images were over 1 MB). We might want to make a small modification later on, but this is good enough for now. No change needed. And thank you for getting the images uploaded.

**Review by stephenfuqua — APPROVED** [review]
Overall: this is good. We may make some further modifications to What's New as we add more to the Model References. There's no reason to hold this up while waiting for more.

File: dataStandard_versioned_docs/version-5/whats-new/whats-new-v52.md (hunk 2 above)
Comment (MrEdData) [reply dataStandard_versioned_docs/version-5/whats-new/whats-new-v52.md:63]: Thanks and sure. I took the liberty to create the diagram image as a JPEG as opposed to BitMap or even GIF. I believe that is the main reason how file size reduced.

File: dataStandard_versioned_docs/version-5/whats-new/whats-new-v52.md (hunk 1 above)
Comment (MrEdData) [reply dataStandard_versioned_docs/version-5/whats-new/whats-new-v52.md:50]: I have started as bold only but the font size in bold was the same with the font size of the header this is part of that is why I have switched to the the underlined and italicized.

**Pull request merged**